corehq/apps/sms/signals.py

```diff
@@ -0,0 +1,17 @@
+from casexml.apps.case.signals import case_post_save
+from casexml.apps.case.models import CommCareCase
+from dimagi.utils.logging import notify_exception
+
+
+def case_changed_receiver(sender, case, **kwargs):
+    try:
+        from corehq.apps.sms.tasks import sync_case_phone_number
+        sync_case_phone_number.delay(case)
```

Comment:
this means that a task of this type will be spawned every time a case is saved.  that seems fine to me since it's better than doing the processing during the actual signal receiver, and in most of the cases it will just be a no-op.